### PR TITLE
About me improvements

### LINKED ISSUE #277: About me improvements
* Increase the max number of characters
* Add some padding to the bottom
* Prevent (unordered) lists on mobile to have their initial dot cut off
* Scroll profile only on about section (i.e. leave tags fixed)

### PR DESCRIPTION
**Pull request by Juggo0**
~So far it only increases the max character limit.~

Closes #277

### CODE REVIEW

**Comment by easafe** [comment]
Then add the padding. I also updated the issue too :clown_face:

**Comment by Juggo0** [comment]
I'm working on `Prevent (unordered) lists on mobile to have their initial dot cut off`, but it looks fine to me?
(Pixel 7 view, all others seem fine too)

image

File: src/Client/css/im.css [hunk 1]
```diff
@@ -86,7 +86,7 @@ body {
 
 .profile-description p:last-of-type,
 .description-message p:last-of-type {
-    margin-bottom: 0px;
+    margin-bottom: 25px;
```
Comment (Juggo0) [inline src/Client/css/im.css:89]: Forgot to mention this in my commit message, but this deals with the `about` section having bottom ~padding~ margin

File: src/Shared/Im/View/SuggestionProfile.purs [hunk 2]
```diff
@@ -239,7 +240,7 @@ displayProfile index loggedUser profileUser temporaryUserMessage =
 badges ∷ ∀ message. Array Badge → Array (Html message)
 badges source = map (it <<< SB.badgeFor) source
       where
-      it bf = HE.div [ HA.class' "badge", HA.title bf.description ] [ HE.img $ [ HA.width "18px", HA.height "18px", HA.class' "badge-img", HA.src $ SR.resourcePath (Left SR.Favicon) Ico ], HE.span [HA.class' "badge-text"] bf.text ]
+      it bf = HE.div [ HA.class' "badge", HA.title bf.description ] [ HE.img $ [ HA.width "18px", HA.height "18px", HA.class' "badge-img", HA.src $ SR.resourcePath (Left SR.Favicon) Ico ], HE.span [ HA.class' "badge-text" ] bf.text ]
```
Comment (Juggo0) [inline src/Shared/Im/View/SuggestionProfile.purs:243]: formatting

**Comment by easafe** [comment]
More good work by iuggo muggo

**Pull request merged**